middlewares/confirm-mail-changePassword.js

```diff
@@ -0,0 +1,52 @@
+const nodemailer = require("nodemailer");
+const User = require("../models/User");
+
+const sendPasswordChangeConfirmationEmail = async (email) => {
+  console.log(email);
+  try {
+    const transporter = nodemailer.createTransport({
+      service: "Gmail",
+      auth: {
+        user: "pruebasdanielmayo@gmail.com",
+        pass: "tfehpymjjkfwsagn",
+      },
+    });
+    let user = await User.findOne({ email });
+    if (!user) {
+      throw new Error("Usuario no encontrado");
+    }
+
+    await transporter.sendMail({
+      from: user.email,
+      to: email,
+      subject: "Confirmación de Cambio de Contraseña",
+      html: `
+        <div style="font-family: Arial, sans-serif; max-width: 600px; margin: 0 auto; padding: 20px; border: 1px solid #dcdcdc; border-radius: 10px;">
```

Comment:
Question... these styles should be in px?

Reply:
Yes, it should go in px, but for now I'm going to consider it as a technical debt to be able to merge it to main